### ISSUE TITLE
passing option returnUri: false and returns uri...

### ISSUE THREAD

**Issue by guitartsword** [feathersjs-ecosystem/feathers-blob#75]
[Similar to this issue](https://github.com/feathersjs-ecosystem/feathers-blob/issues/69)

### Steps to reproduce
use the following config
`{ Model: blobStore, returnUri: false }`

### Expected behavior
```
{
  "id": "filename.zip"
}
```

dont return uri

### Actual behavior
returns uri
```
{
  "id": "filename.zip",
  "uri": "data:application/zip;base64,UEsDBAoAAAAAAHgnfE8AAAAAAAAAAAAAAAAQAAkAYm9vdHN0cmFwLTQuNC4xL1VUBQABxcTfXVBLAwQKAAAACAB4...."
}
```

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):
2.2.0

**NodeJS version**:
v13.1.0

**Operating System**:
ubuntu 18.04

**Comment by claustres** [comment]
Hi, could you give it a try by also setting `returnBuffer: true` as both are exclusive ? Could you also detail the service operation and data input you use to reproduce the bug ? Thanks

**Comment by guitartsword** [comment]
I want `returnBuffer: false` and `returnUri: false`, because I want to generate a Url with amazon s3 and return it

**Comment by claustres** [comment]
I am afraid this behavior is not really implemented, now these options are expected to be mutually exclusive. You can do what you want using a hook that simply removes the data uri form the context and add the required generated Url.

**Comment by guitartsword** [comment]
The only drawback of using a hook is that the server does the **download**, and I want the client to download it directly from a presigned amazon s3 link. I don't see a problem for images and small files, but with big files, the server will **download** the file and to send it will "**upload**"it (if I send the `buffer` or `uri`). This is why I want to only a send a link, for amazon to handle the bandwith directly with the client.

That's why I extended the service and changed the get functionality. I would really like to use `returnUri: false` and `returnBuffer: false` and then use a hook to send the url. This will reduce the bandwith the server uses (No download, no upload).

The only way I found was to extend this service and not call the `super.get` because `super.get` _always_ downloads the file.

**Comment by claustres** [comment]
OK I see, you would like create to upload the file but not get to download it. Of course this module has been designed with symmetric operations in mind, just like a DB adapter saving the object on create and retrieving it on get. Indeed, if get does not download the data it will actually internally returns nothing, which is really strange. In your case you return the URI provided by AWS but it's a specific case not a general one.

I think that overriding the operation is the right way to go, cannot really see what we can do on the module side.

**Comment by cantoute** [comment]
Having same issue...

**Comment by belal-mazlom** [comment]
It should be fixed with this merge #82
